### ISSUE TITLE
[Bug]: Failed to find Build Tools revision 33.0.3

### ISSUE THREAD

**Issue by relyaf** [AndroidIDEOfficial/AndroidIDE#1232]
### What happened?

At first I wanted to take some cool template, but I decided to take an empty "example" so let's say everything went well with the editor, etc. I liked it as soon as I decided to compile this error came out and I can't even understand what the problem is! this was not the case on the previous phone (There was also 12 android, only the number of RAM increased)

### What's the expected behavior?

Successful compilation of an empty clicker application.

### What version of AndroidIDE you're using?

v2.5.2 (debug builds)

### Relevant log output

```shell
> Configure project :app
WARNING: The option setting 'android.aapt2FromMavenOverride=/data/data/com.itsaky.androidide/files/home/.androidide/aapt2' is experimental.
Observed package id 'tools' in inconsistent location '/data/data/com.itsaky.androidide/files/home/android-sdk/cmdline-tools/latest' (Expected '/data/data/com.itsaky.androidide/files/home/android-sdk/tools')

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:compileDebugJavaWithJavac'.
> Could not determine the dependencies of null.
   > Failed to find Build Tools revision 33.0.3

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 11s
```

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by relyaf** [comment]
Please do not close silently! And leave a link if there is such an Issue. Pls

**Comment by MrIkso** [comment]
Download latest build from actions or change build tools version to 33.0.2. Version build tools 33.0.3 is deleted by Google

**Comment by mohammad-ayan-008** [comment]
Same problem #1228

**Comment by relyaf** [comment]
> 

What? How to change tools?

**Comment by BanDroid** [comment]
go to `app/build.gradle`, there's a `buildToolsVersion` specified, yours must be 33.0.3, change it to 33.0.2

**Comment by itsaky** [comment]
This was fixed in #1220.
